src/utils/import_kc_config.sh

```
@@ -15,6 +15,17 @@ KC_CLI_PROJECT_DIR="$TARGET_DIR/keycloak-config-cli"
 KC_CLI_JAR_FILE="keycloak-config-cli.jar"
 REPO_URL="https://github.com/adorsys/keycloak-config-cli.git"
 
+# -----------------------------------------------------------------------------
+# Determine keystore path dynamically
+# -----------------------------------------------------------------------------
+if [[ "$KEYCLOAK_ADMIN_ADDR" == *"localhost"* || "$KEYCLOAK_ADMIN_ADDR" == *"127.0.0.1"* ]]; then
+    KC_KEYSTORE_PATH="$TARGET_DIR/kc_keystore.pkcs12"
+    log "Detected local Keycloak instance. Using keystore path: $KC_KEYSTORE_PATH"
```

I don't fully get this discovery. Don't you need to test the file exists?

---

> I don't fully get this discovery. Don't you need to test the file exists?

I added this to avoid introducing an additional environment variable in `.env`.
The script now supports two possible values for `KC_KEYSTORE_PATH`.

**1. When Keycloak is running locally, the keystore must be located in the target directory during realm import.**

**2. When Keycloak runs in a live environment from the built image, the keystore is located in `/opt/keycloak/target`. This is expected since it is copied there during the image build process.**

To make the setup more robust, I also added a quick check to fail early if the keystore file does not exist, even though this situation should not occur in practice.